utils/vscode/syntaxes/carbon.tmLanguage.json

```diff
@@ -0,0 +1,178 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"name": "carbon",
```

Comment:
Comparing this to the plist syntax, the following things are absent here:

- `foldingStartMarker: "\\{\\s*$", foldingStopMarker: "^\\s*\\}",` -- could we include those here? Is there a reason to not want them?
- `fileTypes: ["carbon"]` -- I don't know what effect this has or if we need it. Do you know?

Reply:
<img width="1008" alt="image" src="https://github.com/carbon-language/carbon-lang/assets/81112205/67c39c0a-0a35-48bb-a20a-31459c8fe485">

Source: https://macromates.com/manual/en/language_grammars you can check the 2<sup>nd</sup> point under Example Grammar